docs/rules/boolean-prop-naming.md

```diff
@@ -1,6 +1,6 @@
-# Enforces consistent naming for boolean props (react/boolean-prop-naming)
+# Enforces consistent naming for boolean props (`react/boolean-prop-naming`)
 
-💼 This rule is enabled in the following [configs](https://github.com/jsx-eslint/eslint-plugin-react#shareable-configurations): `all`.
```

Comment:
The `all` config is no longer mentioned anywhere because I passed the `--ignore-config all` [option](https://github.com/bmish/eslint-doc-generator#configuration-options).